src/cow-react/modules/trade/containers/ImportTokenModal/index.tsx

```diff
@@ -33,19 +35,23 @@ export function ImportTokenModal(props: ImportTokenModalProps) {
   )
 
   // dismiss warning if all imported tokens are in active lists
-  const defaultTokens = useAllTokens()
+  const tokensByAddress = useAtomValue(tokensByAddressAtom)
+  const tokensBySymbol = useAtomValue(tokensBySymbolAtom)
   // example: https://cowswap.dev.gnosisdev.com/#/swap?chain=mainnet&inputCurrency=0xe0b7927c4af23765cb51314a0e0521a9645f0e2a&outputCurrency=0x539F3615C1dBAfa0D008d87504667458acBd16Fa
   const importTokensNotInDefault = useMemo(() => {
     // We should return an empty array until the defaultTokens are loaded
     // Otherwise WETH will be in urlLoadedTokens but defaultTokens will be empty
     // Fix for https://github.com/cowprotocol/cowswap/issues/534
-    if (!Object.keys(defaultTokens).length) return []
+    if (!Object.keys(tokensByAddress).length) return []
```

Comment:
not from this PR, and just nitpicking. But I guess here we are checking for the `length == 0` 
I find it more explicit if we just check that.

Reply:
What if the object is undefined? I prefer this way as it can handle that case as well

Reply:
how can it be undefined?